server/db/index.js

```diff
@@ -3,6 +3,17 @@ var db = require('./_db');
 
 require('./models/user')(db);
 require('./models/products')(db);
+require('./models/orders')(db);
+
+var User = db.model('user');
+var Order = db.model('order');
+var OrderDetail = db.model('orderDetail');
+var Product = db.model('product');
+
+Order.belongsTo(User);
+Order.hasMany(OrderDetail);
+OrderDetail.belongsTo(Order);
+OrderDetail.belongsTo(Product);
```

Comment:
`.belongsTo()` is a 1-1 association, meaning every OrderDetail would have 1 and only 1 product.  Orders will have many Products, and vice versa (though I'm not certain we need that logic), so I think we need to use [`.belongsToMany()`](http://docs.sequelizejs.com/en/latest/docs/associations/#belongs-to-many-associations) here.